### ISSUE TITLE
test_insert_quant_dequant_multi_uses is broken

### ISSUE THREAD

**Issue by mrshenli** [pytorch/pytorch#30465]
https://app.circleci.com/jobs/github/pytorch/pytorch/3723345

```
Nov 26 17:04:42 ======================================================================
Nov 26 17:04:42 ERROR [0.017s]: test_insert_quant_dequant_multi_uses (__main__.TestJit)
Nov 26 17:04:42 ----------------------------------------------------------------------
Nov 26 17:04:42 Traceback (most recent call last):
Nov 26 17:04:42   File "test_jit.py", line 1211, in test_insert_quant_dequant_multi_uses
Nov 26 17:04:42     torch._C._jit_pass_insert_quant_dequant(m._c, "forward", True)
Nov 26 17:04:42 RuntimeError: Method 'get_qparams' is not defined. (get_method at /var/lib/jenkins/workspace/torch/csrc/jit/script/object.h:67)
Nov 26 17:04:42 frame #0: c10::Error::Error(c10::SourceLocation, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) + 0x6c (0x7fd1fb635cdc in /opt/python/3.5/lib/python3.5/site-packages/torch/lib/libc10.so)
Nov 26 17:04:42 frame #1: <unknown function> + 0x310bba9 (0x7fd1feb5dba9 in /opt/python/3.5/lib/python3.5/site-packages/torch/lib/libtorch.so)
Nov 26 17:04:42 frame #2: <unknown function> + 0x30fc0dc (0x7fd1feb4e0dc in /opt/python/3.5/lib/python3.5/site-packages/torch/lib/libtorch.so)
Nov 26 17:04:42 frame #3: <unknown function> + 0x3107882 (0x7fd1feb59882 in /opt/python/3.5/lib/python3.5/site-packages/torch/lib/libtorch.so)
Nov 26 17:04:42 frame #4: torch::jit::InsertQuantDeQuant(torch::jit::script::Module&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, bool) + 0x36 (0x7fd1feb5b976 in /opt/python/3.5/lib/python3.5/site-packages/torch/lib/libtorch.so)
Nov 26 17:04:42 frame #5: <unknown function> + 0x67c59f (0x7fd201cad59f in /opt/python/3.5/lib/python3.5/site-packages/torch/lib/libtorch_python.so)
Nov 26 17:04:42 frame #6: <unknown function> + 0x2703dc (0x7fd2018a13dc in /opt/python/3.5/lib/python3.5/site-packages/torch/lib/libtorch_python.so)
Nov 26 17:04:42 <omitting python frames>
```

cc @ezyang @gchanan @zou3519 @jerryzh168 @suo

**Comment by mrshenli** [comment]
#30470 attempts to fix this

**Comment by mrshenli** [comment]
closed by #30470
